### ISSUE TITLE
Support RTL linked to both .exe and .dll

### ISSUE THREAD

**Issue by ramosian-glider** [ramosian-glider/sanitizers#105]
Originally reported on Google Code with ID 104

```
Problem:
On Windows, each .dll has its own copy of all the statically linked libraries, including
the ASan RTL.
As a result, asan_init is called multiple times in the same address space and the subsequent
executions fail to map the shadow memory.
```

Reported by `timurrrr@google.com` on 2012-08-28 14:35:25

**Comment by ramosian-glider** [comment]
Reported by `timurrrr@google.com` on 2013-07-16 11:43:45
- **Status changed**: `Duplicate`
- **Merged into**: #209

**Comment by ramosian-glider** [comment]
```
Adding Project:AddressSanitizer as part of GitHub migration.
```

Reported by `ramosian.glider` on 2015-07-30 09:12:59
- **Labels added**: ProjectAddressSanitizer
